R/RtGam.R

```diff
@@ -48,15 +52,23 @@
 #' @return Stub function: NULL
 #' @export
 #' @examples
-#' cases <- c(1L, 2L, 3L)
-#' reference_date <- as.Date(c("2023-01-01", "2023-01-02", "2023-01-03"))
-#' mod <- RtGam::RtGam(cases, reference_date)
+#' withr::with_seed(12345, {
```

Comment:
do you really need this?

Reply:
I don't _really_. But I'm throwing a warning whenever the k-index is below one. Under the null that happens 5% of the time. So fitting it to random white noise will randomly throw warnings which I don't love.

Reply:
yeah that makes sense though that high a false positive rate might get pretty annoying pretty fast

Reply:
The joys of NHST